crates/astria-sequencer/src/app/mod.rs

```diff
@@ -433,16 +452,19 @@ impl App {
                     bail!("execution results must be present after executing transactions")
                 };
 
-                self.post_execute_transactions(
-                    process_proposal.hash,
-                    process_proposal.height,
-                    process_proposal.time,
-                    process_proposal.proposer_address,
-                    process_proposal.txs,
-                    tx_results,
-                )
-                .await
-                .wrap_err("failed to run post execute transactions handler")?;
+                let sequencer_block = self
```

Comment:
Unhappy about the code duplication here and at the bottom.

Please create a follow up issue so that we can shift this code around and not have all of this code duplication.